index.html

```diff
@@ -107,14 +156,15 @@ <h4 class="work-h4">Multi-Post Stories</h4>
                             <li class="lang3 bttn">bootstrap</li>
                             <li class="lang4 bttn">ruby</li>
                         </ul>
-                        <button class="project-btn">See Project</button>
+                        <button class="project-btn project-button2">See Project</button>

```

Comment:
- [ ] Kindly, your design should be responsive, as you can see in the image below, the design can be more responsive for not big desktop screens.
- [ ] [OPTIONAL] Add blur background (for the rest of the elements which aren't part of the popup) in your design according to the Figma design you selected.

![image](https://user-images.githubusercontent.com/60164239/131868636-076ea43e-88dc-40aa-ba99-a5c7e7cd54e8.png)

Reply:
- [ ] Kindly add a little bit more details in the CSS styles for the popup in the mobile version to make it more similar to the Figma design you selected.

![image](https://user-images.githubusercontent.com/60164239/131870020-54a5adb2-7ef9-43bb-8a9f-2c414d6740fb.png)